compiler/rustc_privacy/messages.ftl

```diff
@@ -18,7 +18,7 @@ privacy_private_in_public_lint =
     })
 
 privacy_private_interface_or_bounds_lint = {$ty_kind} `{$ty_descr}` is more private than the item `{$item_descr}`
-    .item_note = {$item_kind} `{$item_descr}` is reachable at visibility `{$item_vis_descr}`
+    .item_label = {$item_kind} `{$item_descr}` is reachable at visibility `{$item_vis_descr}`
     .ty_note = but {$ty_kind} `{$ty_descr}` is only usable at visibility `{$ty_vis_descr}`
```

Comment:
These two are a part of the same sentence, they should either both be labels or notes.

Ideally they should be labels, but due to https://github.com/rust-lang/rust/pull/112413#issuecomment-1593916469 we cannot using labels without breaking the order.
So I just suggest keeping notes for now.

Reply:
Ok, due to the same span being used for the main message and the first note the "two notes" scheme looks even worse.